geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/stats/StatWriter.scala

```diff
@@ -24,21 +24,25 @@ import scala.collection.JavaConversions._
 
 trait StatWriter {
```

Comment:
ok for now...but future we may want stats to go elsewhere and have the interface for StatWriter not have ```getStatTable``` since I can envision sending stats a 3rd party API to collect stats...but at the moment we're not even close to that idea but its something to think about in the future